liquibase-core/src/test/java/liquibase/change/AbstractSQLChangeTest.java

```diff
@@ -104,15 +104,15 @@ public void generateCheckSum_changesBasedOnParams() {
 
         ExampleAbstractSQLChange change = new ExampleAbstractSQLChange("SOME SQL");
         change.setSplitStatements(false);
-        assertNotEquals(baseCheckSum.toString(), change.generateCheckSum().toString());
+        assertEquals(baseCheckSum.toString(), change.generateCheckSum().toString());
```

Comment:
Sorry, I missed this question. If I'm not wrong this is because `splitStatements`, `endDelimiter` and `stripComments` are not considered for checksum generation. I don't know if this test makes really sense, but it was already there and I found out it was failing when trying to build the project after performing these changes.

Reply:
No worries! That makes sense that `splitStatements`, `endDelimiter` and `stripComments` do not get included in the checksum calculation. Odd that they were passing prior to this PR, but your changes make sense.